BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Keyword/KeywordFormView.js

```diff
@@ -33,6 +30,22 @@ class KeywordFormView extends mix(AbstractFormView).with(FormViewButtonsMixin)
     }
 
     /**
+     * Render a form
+     *
+     * @private
+     */
+    _renderForm() {
+        super._renderForm();
+        let title = $('#oo_keyword_label', this.$el).val();
+        if (null === this._keywordId) {
+            title = Translator.trans('open_orchestra_backoffice.keyword.title_new');
+        }
+        $('#page-name', this.$el).html(title);
```

Comment:
@alavieille : to avoid to have 2 different mechanisms for setting the title. It could have been possible to set via template the defaut title (new xxxx) and overrride it on edition via jquery, but it could lead sometimes (when buggued) to display 'new xxxx' on an edit form